### ISSUE TITLE
3 tests fail on macOS 10.6.8 (apparently due to NA being quoted/non-quoted)

### ISSUE THREAD

**Issue by barracuda156** [jmbarbone/mark#187]
```
R version 4.3.0 (2023-04-21) -- "Already Tomorrow"
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: powerpc-apple-darwin10.8.0 (32-bit)

> library(testthat)
> library(mark, warn.conflicts = FALSE)
> 
> test_check("mark")
Starting 4 test processes
[ FAIL 3 | WARN 0 | SKIP 4 | PASS 808 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• Is not interactive (1)
• getRversion() >= "4.3" is TRUE (1)
• not interactive: check DESCRIPTION date (1)
• not static (1)

══ Failed tests ════════════════════════════════════════════════════════════════
── Failure ('test-clipboard.R:59:3'): utils_type_convert() ─────────────────────
type_convert2(xchr) (`actual`) not equal to `rchr` (`expected`).

`actual[5:8]`:   "them" ".011" "2020" NA  
`expected[5:8]`: "them" ".011" "2020" "NA"
── Failure ('test-switch.R:34:3'): switch_in_case() works as expected ──────────
`res` (`actual`) not equal to `exp` (`expected`).

`actual`:   "a" "a" "NA" "e" "NA" "NA"
`expected`: "a" "a" NA   "e" NA   NA  
── Failure ('test-switch.R:58:3'): switch_in_case() handles evaluations ────────
`res` (`actual`) not equal to `exp_res` (`expected`).

`actual[2:8]`:   "a" "b" "b" "NA" "d" "d" "d"
`expected[2:8]`: "a" "b" "b" NA   "d" "d" "d"

[ FAIL 3 | WARN 0 | SKIP 4 | PASS 808 ]
Error: Test failures
Execution halted
```

**Comment by jmbarbone** [comment]
These should have fixed with in #182 (which will be part of #185)

@barracuda156, can you confirm if you're still seeing this in the current dev version?

```r
remotes::install_github("jmbarbone/mark")`
```

**Comment by barracuda156** [comment]
> These should have fixed with in #182 (which will be part of #185)
> 
> @barracuda156, can you confirm if you're still seeing this in the current dev version?
> 
> ```r
> remotes::install_github("jmbarbone/mark")`
> ```

@jmbarbone Thank you, 0.6.0 passes all tests fine.
